### ISSUE TITLE
"builder": "simple" doesn't work any more

### ISSUE THREAD

**Issue by swun90** [SublimeText/LaTeXTools#1050]
I used ST3 in windows for years. Last week the `"builder":"simple"` works well, running `pdflatex`, `bibtex`, `pdflatex`, `pdflatex` automatically. But now it goes wrong. It doesn't show the console, which displayed only the warnings and errors. Instead, it only shows "building" in the status bar.
Yesterday it showed all the log information in the console.

I changes the builder to traditional, reopen ST3, but nothing different.
I don't know what'r wrong.
Any suggestions?

**Comment by r-stein** [comment]
Does it work for other documents? You can also switch the builders with ctrl+shift+b

**Comment by swun90** [comment]
No. I tried another TeX file. Even though `"builder"` in settings `LaTeXTools.sublime-settings` is `simple`, it built with `traditional` way, showing the console and displaying `TraditionalBuilder`. I guess it's because I used `traditional` way last time.
If trying Ctrl+Shift+B, choosing builder A, it will build with builder A. Next time pressing Ctrl+B, it will build with builder A. If trying C+S+B and builder B, next builder B will be used.

But this is not so good as previous way. Why does the settings `builder` not work?
